content/md/productivity/manage-your-rules.md

```diff
@@ -92,21 +99,35 @@ Depending on the action you choose, the template changes. The table below lists
 
 #### Available actions in the UI
 
-| Actions  | What they do |
+| Set actions  | What they do |
 |---|---|
 | Set categories  | Classify your products/product models in the categories you choose, and **remove** them from their former categories.  |
-| Add categories  | Add your products/product models to new categories, **without removing** them from their former categories.  |
 | Set family  | Replace your products/product models family |
+| Set attribute value  | Set values to a multi select, simple select, number, text, textarea, date, price collection, asset collection or boolean attribute. It will remove the former values |
```

Comment:
```suggestion
| Set attribute value  | Set values to a multi select, simple select, number, text, textarea, date, price collection, asset collection or boolean attribute. It will replace the former values |
```